packages/editor-ui/src/store.ts

```diff
@@ -470,6 +476,11 @@ export const store = new Vuex.Store({
 
 			state.stateIsDirty = true;
 			Vue.set(node, 'parameters', updateInformation.value);
+
+			if (!state.nodeMetadata[node.name]) {
+				Vue.set(state.nodeMetadata, node.name, {});
+			}
+			Vue.set(state.nodeMetadata[node.name], 'parametersLastUpdatedAt', new Date().getTime());
```

Comment:
you can also use this for numeric timestamps 👍🏼 

```suggestion
			Vue.set(state.nodeMetadata[node.name], 'parametersLastUpdatedAt', Date.now());
```

Reply:
nice